core/probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/Actions.scala

```diff
@@ -1,27 +1,32 @@
 package org.virtuslab.ideprobe.handlers
 
-import com.intellij.openapi.actionSystem.ActionManager
-import com.intellij.openapi.actionSystem.ActionPlaces
-import com.intellij.openapi.actionSystem.AnAction
-import com.intellij.openapi.actionSystem.AnActionEvent
-import com.intellij.openapi.actionSystem.CommonDataKeys
-import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.actionSystem.{
+  ActionManager,
+  ActionPlaces,
+  AnAction,
+  AnActionEvent,
+  CommonDataKeys,
+  DataContext
```

Comment:
I'm not sure this is proper formatting.

Reply:
It's done with intellij's "reformat code" action